scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java

```diff
@@ -236,37 +228,33 @@ private boolean isRowSelected(int rowNum) {
         ListGridField progressField = fields.get(PROGRESS_ATTR);
         progressField.setType(ListGridFieldType.FLOAT);
         progressField.setAlign(Alignment.CENTER);
-        progressField.setCellFormatter(new CellFormatter() {
-            public String format(Object value, ListGridRecord record, int rowNum, int colNum) {
-                int pw = getFieldWidth(PROGRESS_ATTR.getName());
-                float progress = 0;
-                if (value != null) {
-                    progress = Float.parseFloat(value.toString());
-                }
-                int bx = new Double(Math.ceil(pw * progress)).intValue() - 601;
-                String progressUrl = SchedulerImages.instance.progressbar().getSafeUri().asString();
-
-                String style = "display:block; " + //
-                               "border: 1px solid #acbac7; " + //
-                               "background-image:url(" + progressUrl + ");" + //
-                               "background-position:" + bx + "px 0px;" + //
-                               "background-repeat: no-repeat;" + //
-                               "background-color:#a7cef6";
-
-                Job job = JobRecord.getJob(record);
-                String progressCounters = job.getFinishedTasks() + " / " + job.getTotalTasks();
-                return "<div style='" + style + "'>" + progressCounters + "</div>";
+        progressField.setCellFormatter((value, record, rowNum, colNum) -> {
+            int pw = getFieldWidth(PROGRESS_ATTR.getName());
+            float progress = 0;
+            if (value != null) {
+                progress = Float.parseFloat(value.toString());
             }
+            int bx = new Double(Math.ceil(pw * progress)).intValue() - 601;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this "Double" constructor [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2129)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the boxing to "Double". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2153)